### ISSUE TITLE
IPA analysis error: AttributeError: 'str' object has no attribute 'get'

### ISSUE THREAD

**Issue by luminescence69** [MobSF/Mobile-Security-Framework-MobSF#1885]
## ENVIRONMENT

```
OS and Version: Kali GNU / 2021.4
Python Version: Python 3.9.9
MobSF Version: v3.4.6 Beta
```

## EXPLANATION OF THE ISSUE

```
I upload an IPA to MobSF, expect the report, get an immediate crash due to the following bug:

`[INFO] 13/Jan/2022 08:55:41 - MIME Type: application/octet-stream FILE: Helios.i.6.0.0.bat.ipa                                                            
[INFO] 13/Jan/2022 08:55:41 - Performing Static Analysis of iOS IPA
[INFO] 13/Jan/2022 08:55:41 - iOS Static Analysis Started
[INFO] 13/Jan/2022 08:55:41 - iOS Binary (IPA) Analysis Started
[INFO] 13/Jan/2022 08:55:41 - Generating Hashes
[INFO] 13/Jan/2022 08:55:41 - Extracting IPA
[INFO] 13/Jan/2022 08:55:41 - Unzipping
[INFO] 13/Jan/2022 08:55:42 - Get Files, BIN Plist -> XML, and Normalize
[WARNING] 13/Jan/2022 08:55:42 - Failed to convert plist
[INFO] 13/Jan/2022 08:55:42 - iOS Info.plist Analysis Started
[INFO] 13/Jan/2022 08:55:42 - Finding Info.plist in iOS Binary
[INFO] 13/Jan/2022 08:55:42 - Checking Permissions
[INFO] 13/Jan/2022 08:55:42 - Checking for Insecure Connections
[INFO] 13/Jan/2022 08:55:42 - Fetching Details from App Store: tma.com.dhl.test                                                                           
[WARNING] 13/Jan/2022 08:55:42 - Unable to get app details.
[INFO] 13/Jan/2022 08:55:42 - Starting Binary Analysis
[INFO] 13/Jan/2022 08:55:42 - Running MachO Analysis on DHL eCommerce
[INFO] 13/Jan/2022 08:55:43 - Getting Binary Information
[INFO] 13/Jan/2022 08:55:43 - Dumping classes
[INFO] 13/Jan/2022 08:55:43 - Running jtool against the binary for dumping classes                                                                        
[INFO] 13/Jan/2022 08:55:43 - Running strings against the Binary
[INFO] 13/Jan/2022 08:55:43 - Fetching icon path
[INFO] 13/Jan/2022 08:55:43 - Starting IPA URL and Email Extraction
[INFO] 13/Jan/2022 08:55:45 - Performing Malware Check on extracted Domains
[INFO] 13/Jan/2022 08:55:45 - Maltrail Database is up-to-date
[INFO] 13/Jan/2022 08:55:49 - Finished URL and Email Extraction
[INFO] 13/Jan/2022 08:55:49 - Detecting Firebase URL(s)
[INFO] 13/Jan/2022 08:55:49 - Connecting to DB
[INFO] 13/Jan/2022 08:55:49 - Saving to Database
[ERROR] 13/Jan/2022 08:55:49 - Error Performing Static Analysis
Traceback (most recent call last):
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/mobsf/StaticAnalyzer/views/ios/static_analyzer.py", line 185, in static_analyzer_ios
    return render(request, template, context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 176, in render
    return self._render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 168, in _render
    return self.nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 153, in render
    return compiled_parent._render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 168, in _render
    return self.nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/defaulttags.py", line 295, in render
    return nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 1038, in render
    output = self.filter_expression.resolve(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 728, in resolve
    new_obj = func(obj, *arg_vals)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/mobsf/StaticAnalyzer/views/android/static_analyzer.py", line 79, in key
    return data.get(key_name)
AttributeError: 'str' object has no attribute 'get'
[ERROR] 13/Jan/2022 08:55:49 - 'str' object has no attribute 'get'
[ERROR] 13/Jan/2022 08:55:49 - Internal Server Error: /static_analyzer_ios/`

```

## STEPS TO REPRODUCE THE ISSUE

```
1. Cloned latest Git version.
2. Execute setup
3. Execute run.sh
4. Upload IPA
```

**Comment by ajinabraham** [comment]
Please share the IPA for reproducing this.

**Comment by luminescence69** [comment]
> Please share the IPA for reproducing this.

Can I email it to you or is there a secure file share?

**Comment by ajinabraham** [comment]
Yes, you can email at (ajin25) Gmail or PM over slack.

**Comment by ajinabraham** [comment]
Thanks for sharing the binary, reproduced this.

**Comment by ajinabraham** [comment]
This will be fixed in 3.5.0
